Cargo.toml

```diff
@@ -31,9 +31,11 @@ futures-util = { version = "0.3", default-features = false }
 tokio = { version = "1.0", features = ["io-util", "net"] }
 either = "1"
 thiserror = "1.0"
+async-trait = "0.1.81"
```

Comment:
I guess GSS-API is relatively less common. Can you put GSS-API support under a feature so people not needing it won't pull this extra dependency?

Reply:
Yep will do that